server/database/schema.sql

```diff
@@ -2,14 +2,14 @@ create table user (
   id int unsigned primary key auto_increment not null,
   firstname varchar(80) not null,
   lastname varchar(80) not null,
-  city varchar(80),
+  city varchar(80) not null,
   email varchar(255) not null unique,
   hashed_password varchar(255) not null,
   image text,
   admin boolean default 0
```

Comment:
pourquoi l'admin n'as pas de not null ??

ça me parait  problématique si un user n'a pas de role ^^